### DISCUSSION TITLE
Accessibility question about external Link  in the Link component

### DISCUSSION THREAD

**Discussion by BoyanLiuu** [cloudscape-design/components#366] [Q&A] [answered]
Hello friends,
I would like to ask one accessibility question for the External Link example in the Link component
This is external link , so it contains default icon and it will open in a new window page.

Here is the code I found from document:
```
import * as React from "react";
import Link from "@cloudscape-design/components/link";

export default () => {
  return (
    <Link
      external
      externalIconAriaLabel="Opens in a new tab"
      href="https://example.com/"
    >
      Learn more
    </Link>
  );
}
```

<img width="300" alt="image" src="https://user-images.githubusercontent.com/45325025/195483070-7b4d4426-5246-4863-a0e6-4a900ea54352.png">

### My question is do we need to add both externalIconAriaLabel and ariaLabel ?
Here is what I found out:

In the Usage section ,  I find out that I have to use ariaLabel , because the code would open in a new window page.
<img width="910" alt="image" src="https://user-images.githubusercontent.com/45325025/195483421-c2091e1a-44eb-4b45-a6cd-018759dd76e1.png">

However i am not sure if we need to add externalIconAriaLabel prop.
I inspect the use case in the [documentation ](https://cloudscape.design/about/connect/). It seems like we did not add externalIconAriaLabel.

Could anyone help me confirm  whether we should use both prop or not?  Thanks for the help! 
 
<img width="920" alt="image" src="https://user-images.githubusercontent.com/45325025/195483827-63aaa1f4-8d32-4000-a403-6f842d0ff7ed.png">

**Comment by FlorianDr** [comment, marked as answer]
For external link, you only need to provide a label for the icon using `externalIconAriaLabel`. It should be a text similar to "Opens in a new tab".
